spec/index.bs

```diff
@@ -4738,17 +4738,17 @@ interface GPUCommandEncoder {
     undefined copyBufferToTexture(
         GPUImageCopyBuffer source,
         GPUImageCopyTexture destination,
-        GPUExtent3D copySize);
+        optional GPUExtent3D copySize = {});
 
     undefined copyTextureToBuffer(
         GPUImageCopyTexture source,
         GPUImageCopyBuffer destination,
-        GPUExtent3D copySize);
+        optional GPUExtent3D copySize = {});
```

Comment:
To be honest, I'd rather have the `width` of the extent be a required member. There is no use for having it a default here, it's just more confusing from the API standpoint.

Reply:
That would also fix the problem from the Web IDL perspective. Should just `width` be required, or `height` as well?

Reply:
nah, I think height is fine to be optional, but width is something we expect everybody to specify.

Reply:
I've sent https://github.com/gpuweb/gpuweb/pull/1441 to do that. I'm not sure if there's prose that also needs adjusting though.